tabular/src/autogluon/tabular/models/catboost/callbacks.py

```diff
@@ -139,7 +139,13 @@ def __init__(self, stopping_rounds, eval_metric, compare_key='validation'):
 
     def after_iteration(self, info):
         is_best_iter = False
-        cur_score = info.metrics[self.compare_key][self.eval_metric_name][-1]
+        if self.eval_metric_name.startswith('MultiQuantile'):
+            # FIXME: CatBoost adds extra ',' in the metric name if quantile levels are not balanced
+            # e.g., 'MultiQuantile:alpha=0.1,0.25,0.5,0.95' becomes 'MultiQuantile:alpha=0.1,,0.25,0.5,0.95'
+            eval_metric_name = [k for k in info.metrics[self.compare_key] if k.startswith('MultiQuantile')][0]
```

Comment:
would it be better to do `k.startswith('MultiQuantile:')`

Reply:
make the prefix ('MultiQuantile') a variable, re-use whenever the string is used in the code.

Reply:
I defined `CATBOOST_QUANTILE_PREFIX = 'MultiQuantile:'` (with the colon) to avoid potential collisions with `MultiQuantile` to be extra safe.